pkg/ingress/gateway.go

```diff
@@ -22,61 +21,14 @@ import (
 //    for it and stores it in the referenced secret.
 // 2. Starts a goroutine to watch for changes to the MeshConfig resource and certificate rotation, and
 //    updates/removes the certificate and secret as necessary.
-func (c *client) provisionIngressGatewayCert(stop <-chan struct{}) error {
-	defaultCertSpec := c.cfg.GetMeshConfig().Spec.Certificate.IngressGateway
-	if defaultCertSpec != nil {
-		// Issue a certificate for the default certificate spec
-		if err := c.createAndStoreGatewayCert(*defaultCertSpec); err != nil {
-			return errors.Wrap(err, "Error provisioning default ingress gateway cert")
-		}
-	}
+func (c *client) provisionIngressGatewayCert(stop <-chan struct{}) {
```

Comment:
There was a bug here before, where a rotation could occur prior to the subscription starting, which would be missed.